### PR TITLE
Abracadabra: Fix new deployment

### PR DESCRIPTION
**Pull request by nemani**
Fixes a couple of bugs in abracadabra code, which makes sure that the subgraph can be deployed again for mainnet.

### CODE REVIEW

**Comment by nemani** [comment]
@tannishmango @harsh9200 @0xbe1 @this-username-is-taken

**Review by 0xbe1 — COMMENTED** [review]
lgtm

File: subgraphs/abracadabra/src/common/getters.ts [hunk 1]
```diff
@@ -194,18 +208,26 @@ export function getOrCreateMarketDailySnapshot(
...
+    marketMetrics.dailyRepayUSD = BIGDECIMAL_ZERO;
 
-    marketMetrics.cumulativeTotalRevenueUSD = BIGDECIMAL_ZERO;
-    marketMetrics.cumulativeSupplySideRevenueUSD = BIGDECIMAL_ZERO;
-    marketMetrics.cumulativeProtocolSideRevenueUSD = BIGDECIMAL_ZERO;
-
+    marketMetrics.rates = [];
+    marketMetrics.blockNumber = event.block.number;
+    marketMetrics.timestamp = event.block.timestamp;
+    marketMetrics.save();
```
[the remark this answers is not in the document]
Comment (nemani) [reply subgraphs/abracadabra/src/common/getters.ts:230]: Noticed and fixed!

**Comment by this-username-is-taken** [comment]
Thanks guys!

**Pull request merged**